src/styles/blocks/caledar.scss

```diff
@@ -0,0 +1,69 @@
+$grey: #eee;
+$squere-side: 100px;
+$calendar-gap: 1px;
+$calendar-day-border: 1px;
+$calendar-day-space: $squere-side + $calendar-gap;
+$calendar-padding: 10px;
+$calendar-widht: ($squere-side + $calendar-gap) * 7;
+
+.calendar {
+  width: $calendar-widht;
+  margin: 0 auto;
```

Comment:
The calendar should be centered both horizontally and vertically. Maybe try to add display flex to parent, which is body